src/Subtitles/index.js

```diff
@@ -75,6 +76,27 @@ export default {
     const subtitles = getOrCreateSubtitlesComponent()
     subtitles.textAlignVertical = v
   },
+  viewportW(v) {
+    const subtitles = getOrCreateSubtitlesComponent()
+    subtitles.viewportW = v
+  },
+  viewportH(v) {
+    const subtitles = getOrCreateSubtitlesComponent()
+    subtitles.viewportH = v
+  },
+  viewport(w, h) {
+    const subtitles = getOrCreateSubtitlesComponent()
+    subtitles.viewportW = w
+    subtitles.viewportH = h
+  },
+  xPos(v) {
+    const subtitles = getOrCreateSubtitlesComponent()
+    subtitles.xPos = v
+  },
+  yPos(v) {
+    const subtitles = getOrCreateSubtitlesComponent()
+    subtitles.yPos = v
+  },
```

Comment:
wouldn't it be more intuitive to have only one method for this that accepts an object with 4 keys (width, height, x and y)?

Reply:
From a frontend point of view I really like the individual value method, this allows you to change the values a bit more dynamically during run time without you having to know f.e. the width of the viewport. While using methods with more parameters are handy during the initialisation phase of the player.
The individual value methods are also very handy when using the `styles` method, this allows you to do: 

Subtitles.styles({
  xPos: 'center',
  fontSize: 40,
  viewportH: 540
})